themes/hugo-planetarium/assets/scss/style.scss

```diff
@@ -43,6 +43,25 @@ blockquote p {
   display: inline;
 }
 
+blockquote > p:last-child {
+  cite:only-child { display: block; margin-top: 1em; }
+  cite:before { content: '\2014'; } // em dash
```

Comment:
Kindly reminder that the `content` will not be read to screen readers: https://developer.mozilla.org/en-US/docs/Web/CSS/content#accessibility_concerns

Although I *think* it is fine since it is a punctuation.

Reply:
You're right.  That's why I always use `content:` prop for punctuations only.